src/components/Nav/Nav.scss

```
@@ -0,0 +1,59 @@
+.navContainer {
+  display: flex;
+  margin: 20px 45px;
+
+  .navBox {
+    display: flex;
+    position: relative;
+    justify-content: center;
+    height: 95px;
+    width: 100%;
+    margin-bottom: 10px;
+  }
+
+  .navLogo {
+    width: 140px;
+    height: 74px;
+  }
+
+  .topUl {
+    position: fixed;
+    display: flex;
+    right: 45px;
+  }
+
+  .topLi {
+    list-style-type: none;
+    margin-left: 40px;
+    font-size: 18px;
+  }
+
+  .topLink {
+    color: black;
+  }
+
+  .menu {
+    position: absolute;
+    display: flex;
+    align-items: flex-end;
+    width: 200px;
+    height: 300px;
+    // border: 1px solid lightgray;
+  }
+
+  .searchBox {
+    width: 100%;
+    border-bottom: 1px solid black;
+  }
+
+  .search {
+    width: 100%;
+    font-size: 100%;
+    border: none;
+  }
+
+  .search::placeholder {
+    color: black;
+    font-size: 20px;
+  }
```

```suggestion
  .search {
    width: 100%;
    border: none;
    
    &::placeholder {
      color: black;
      font-size: 20px;
    }
```
- font-size: 100% 가 속성을 주지 않은 기본 값과 다른가요?
- & 연산자 활용할 수 있는 부분 같네요!